contrib/wmenu/template.py

```diff
@@ -0,0 +1,27 @@
+pkgname = "wmenu"
+pkgver = "0.1.5"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = [
+    "meson",
+    "pkgconf",
+    "scdoc",
+]
+makedepends = [
+    "cairo",
+    "libxkbcommon-devel",
+    "pango-devel",
+    "wayland-devel",
+    "wayland-protocols",
+]
+depends = ["cairo", "pango", "wayland", "libxkbcommon"]
```

Comment:
```suggestion
```

these are all traced

Reply:
the non-devel cairo in makedepends is also incorrect probably

Reply:
NEEDED entries in elfs are added as dependencies automatically via `so:libcairo.so.2` or whatnot, you don't need explicit library depends= usually

Reply:
that's very cool, i've removed the depends now